cli/flags.rs

```diff
@@ -221,8 +221,10 @@ To evaluate code in the shell:
 
 lazy_static! {
   static ref LONG_VERSION: String = format!(
-    "{}\nv8 {}\ntypescript {}",
+    "{} ({})\ntarget {}\nv8 {}\ntypescript {}",
     crate::version::DENO,
+    crate::version::BUILD,
+    env!("TARGET"),
```

Comment:
It's a little odd that the target is on its own line. Maybe it should be in the parens too?

Reply:
How about this format?

```shellsession
$ deno --version
deno 1.4.6 (0dcdc5e, x86_64-apple-darwin)
v8 8.7.220.3
typescript 4.0.3
```

Reply:
I think that makes more sense. It would be nice to also know if we're using release/debug build.